### PR TITLE
[HDFS Reader] Use timestamp of the current day as the default value for latest_committed_timestamp

### PR DESCRIPTION
**Pull request by Willymontaz**
Details: we faced the case of constant failures during file committing, that induced restart of reader instances
Those restart led to think that latest committed timestamp was making progress, which was fake.

Using a constant value for the day, across restart will help to catch such issue.

### CODE REVIEW

File: readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java [hunk 1]
```diff
@@ -196,10 +197,9 @@ private double getLatestCommittedTimestamp() {
         //there are cases for which we won't find a value for the latest committed timestamp
         // - the first time this code goes in, no file has the correct metadata
         // - for a new event type, we have no history too, so no value
-        //By using the default value 'now' rather than 0, we prevent firing unnecessary alerts
-        //However, if there is an actual problem and the reader never commits, it will eventually fire
-        //an alert.
-        long defaultValue = System.currentTimeMillis();
+        // - if started and no file has been create for the day
```
Comment (gray-matter) [inline readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java]: "created"

File: readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java [hunk 2]
```diff
@@ -196,10 +197,9 @@ private double getLatestCommittedTimestamp() {
...
+        //Using the timestamp of today midnight prevents unnecessary alerts
+        long defaultValue = LocalDateTime.now().toLocalDate().atStartOfDay().toEpochSecond(ZoneOffset.UTC) * 1000;
```
Comment (mk-dhia) [inline readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java]: Why we need a changing name. maybe just make sure that alerting do not use the file name as source of information. and use a constant name that at least will be clear that file is created because there is nothing to commit

Comment (gray-matter) [reply readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java]: Not sure where you see a file name. This is the timestamp for the monitoring/alerting part, it's not used for files naming (cf tests).

**Comment by Willymontaz** [comment]
The new default value is not great either. Maybe we should just put something like 0 so we can clearly identified this corner case. I prefer having a false alert once in a while (should be only for new events and first time the commit was deployed) that disappears at the first file correctly committed or stays if there is really an issue.
